packages/design-tokens/tokens/color/border.json

```diff
@@ -15,30 +15,20 @@
       "button": {
         "primary": {
           "success": {
-            "base": { "value": "{color.base.green.value}" },
-            "hover": { "value": "{color.base.green.value}" },
-            "active": { "value": "{color.core.green.4.value}" },
-            "focused": { "value": "{color.core.green.4.value}" },
-            "disabled": { "value": "{color.base.green.value}" }
+            "base, hover": { "value": "{color.base.purple.value}" },
+            "hover, active, focused": { "value": "{color.core.purple.4.value}" },
+            "disabled": { "value": "{color.base.purple.2.value}" }
```

Comment:
`color.base.purple.2` doesn't exist, perhaps you meant `color.core.purple.2.value`?